app/src/main/java/ch/epfl/sdp/item/Healthpack.java

```diff
@@ -16,6 +16,11 @@ public Healthpack(double healthPackAmount) {
         this.healthPackAmount = healthPackAmount;
     }
 
+    @Override
+    public Item createCopy() {

```

Comment:
Why do you need to have a "createCopy" method  when you could have used obj.clone() ? I am not saying what you did is wrong, I am just curious why you chose this approach. The advantage I see with the Cloneable interface is that you might be able to make "Items.java" implement Cloneable and then override the clone method in each item class (that is the healthpack, the shrinker, the trap, ...).

Also, given that you return a healthpack, I would suggest renaming the method to "createHealthpackCopy()".

Reply:
Totally fine with this, I just forgot that clonable exists, I will quickly fix this